kyo-scheduler-zio/jvm/src/main/scala/kyo/KyoSchedulerRuntime.scala

```diff
@@ -0,0 +1,31 @@
+package kyo
+
+import kyo.scheduler.*
+import zio.*
+import zio.internal.ExecutionMetrics
+import zio.internal.FiberRuntime
+
+object KyoSchedulerRuntime {
+
+    lazy val default: Runtime[Any] = {
+        val exec =
+            new Executor {
+                val scheduler =
+                    kyo.scheduler.Scheduler.get
+                def metrics(implicit unsafe: Unsafe): Option[ExecutionMetrics] = None
+                def submit(runnable: Runnable)(implicit unsafe: Unsafe): Boolean = {
+                    scheduler.schedule(kyo.scheduler.Task(runnable.run()))
+                    true
+                }
+            }
+        Unsafe.unsafe { implicit u =>
+            Runtime.default.unsafe.run {
+                for {
+                    scope   <- Scope.make
+                    env     <- (Runtime.setExecutor(exec) ++ Runtime.setBlockingExecutor(exec)).build(scope)
+                    runtime <- ZIO.runtime[Any].provideEnvironment(env)
+                } yield runtime
+            }.getOrThrowFiberFailure()
```

Comment:
I found this approach that avoids the regression reported in https://github.com/zio/zio/pull/8869. I'm not sure about the implications of how it manages the scope but it might be ok since Kyo's default scheduler isn't meant to be shut down.